fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -2317,9 +2382,9 @@ private static String getUrlPath(Entry entry) {
      * @return local reference map
      */
     private Map<String, String> buildLocalRefMap(Bundle requestBundle, Map<Integer, Entry> validationResponseEntries) throws Exception {
-            Map<String, String> localRefMap = new HashMap<>();
+        Map<String, String> localRefMap = new HashMap<>();
 
-        for (int entryIndex=0; entryIndex<requestBundle.getEntry().size(); ++entryIndex) {
+        for (int entryIndex = 0; entryIndex < requestBundle.getEntry().size(); entryIndex++) {
```

Comment:
so did we have an off-by-one error before?

Reply:
i guess either one is fine, so its really just a style thing

Reply:
I just updated it for consistency with the coding style of the rest of the class.